app/controllers/githook_controller.rb

```diff
@@ -27,6 +30,7 @@ def master
     render :index
   end
 
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.